plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2133,6 +2135,23 @@ public void testProjectionPushdownMultipleRows()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPartitionFilterIncluded()
+    {
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty(getSession().getCatalog().orElseThrow(), "query_partition_filter_required", "true")
+                .build();
+
+        try (TestTable table = new TestTable(
+                new TrinoSqlExecutor(getQueryRunner(), session),
```

Comment:
nit: We can replace it as `getQueryRunner::execute` as this property doesn't affect INSERT/CREATE.